test_launch_ros/test/test_launch_ros/frontend/test_find_frontend.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test find-package-share substitution."""
+
+import io
+import textwrap
+from typing import List
+from typing import Optional
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import Action
+from launch import LaunchContext
+from launch import LaunchDescriptionEntity
+from launch import LaunchService
+from launch import SomeSubstitutionsType
+from launch.frontend import expose_action
+from launch.frontend import Parser
+from launch.utilities import normalize_to_list_of_substitutions
+from launch.utilities import perform_substitutions
+
+import pytest
+
+
+@expose_action('test_find_frontend')
+class CheckSubstitutionAction(Action):
+
+    def __init__(self, subst: SomeSubstitutionsType, result: SomeSubstitutionsType, **kwargs):
+        self.subst = normalize_to_list_of_substitutions(subst)
```

Comment:
I guess the issue I'm running into is because `ExecuteProcess` doesn't call `normalize_to_list_of_substitutions`.

Reply:
See my reply [here](https://github.com/ros2/launch_ros/issues/74#issuecomment-533257827).

Reply:
And `ExecuteProcess` does call `normalize_to_list_of_substitutions`: https://github.com/ros2/launch/blob/a2f3255aa7a9823fbf330269f170d1350d8a0378/launch/launch/actions/execute_process.py#L203.